auth-web/src/util/http-util.ts

```diff
@@ -0,0 +1,23 @@
+import ConfigHelper from 'sbc-common-components/src/util/config-helper'
```

Comment:
we can use the ConfigHelper from our app itself

Reply:
haaa..stupid IDE suggested i